crates/core/tedge_mapper/src/c8y/tests.rs

```diff
@@ -995,12 +992,12 @@ async fn c8y_mapper_syncs_pending_child_alarms_on_startup() {
 
     sm_mapper.abort();
 }
+*/
 
-#[tokio::test(flavor = "multi_thread", worker_threads = 2)]
+#[tokio::test]
```

Comment:
General question to us:
Now shouldn't we use "multi_thread" in tests? Why did we change?